### ISSUE TITLE
Remove 'compass' and 'susy' from mandatory dependencies

### ISSUE THREAD

**Issue by barraq** [barraq/pandoc-moderncv#4]
Currently you need compass and susy to build your CV will it is only required when customising the CV look and feel. To prevent this, the generated css stylesheet should embedded into the release package.

**Comment by carliros** [comment]
I agree with that, it will be great to remove dependencies to ruby libs (compass and susy), so that it is easy to try it out.  Furthermore, that can be a feature for experts ;) and for common users it could be enough to just modify the markdown and generate the CV with pandoc.
 I wonder if this enhancement is already implemented ...
